test/adagrams.test.js

```diff
@@ -120,7 +120,8 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+      // throw "Complete test";
+      expectScores({});
```

Comment:
It looks like this test was not actually completed. Please make sure you complete all of the tests!